core/primitives/src/genesis/block.rs

```diff
@@ -45,7 +45,7 @@ impl Block {
                 initial_total_supply,
                 compute_bp_hash_from_validator_stakes(
                     validator_stakes,
-                    ProtocolFeature::BlockHeaderV3.enabled(genesis_protocol_version),
+                    true, // BlockHeaderV3 is deprecated and always enabled
```

Comment:
nit: The other comment 

`// We always use use_versioned_bp_hash_format after BlockHeaderV3 feature`

was clearer than this